Makefile

```diff
@@ -177,4 +177,4 @@ travis-slack-reporter:
   	  	--env TRAVIS_PULL_REQUEST=$(TRAVIS_PULL_REQUEST) \
    		--env TRAVIS_BRANCH=$(TRAVIS_BRANCH) \
 		--env TRAVIS_BUILD_WEB_URL=$(TRAVIS_BUILD_WEB_URL) \
-		quay.io/open-cluster-management/grc-ui-tests:latest node ./tests/utils/slack-reporter.js
+		quay.io/open-cluster-management/grc-ui-tests-2.3 node ./tests/utils/slack-reporter.js
```

Comment:
This doesn't look right...it's missing the `:latest` part

Reply:
Good catch